w3-store.md

```diff
@@ -1,631 +1,870 @@
 
-`store/remove` can be derived from a [`store/*`](#store) or [`*`][ucan-spec-top] capability with a matching `with` field.
+##### Store Remove IPLD Schema
 
-#### Caveats
+```ipldsch
+type StoreRemoveCapability struct {
+  with      SpaceDID
+  nb        StoreRemove
+}
 
-When invoking `store/remove`, the `link` caveat must be set to the CID of the CAR file to remove.
+type StoreRemove struct {
+  link            &ContentArchive
+}
+```
 
-If a delegation contains a `link` caveat, an invocation derived from it must have the same CAR CID in its `link` field. A delegation without a `link` caveat may be invoked with any `link` value.
+##### Store Remove Content
 
-| `field`   | `value`                           | `required?` | `context`                                     |
-| --------- | --------------------------------- | ----------- | --------------------------------------------- |
-| `link` | CAR CID string, e.g. `bag...`     | ✔           | The CID of the CAR file to remove.            |
+Capability invocation MUST specify desired [Content archive Identifier].
 
-#### Invocation
+##### Store Remove Example
 
 ```js
 {
-  can: "store/remove",
-  with: "did:key:abc...",
-  nb: {
-    link: "bag...",
+  "v": "0.9.1",
+  "iss": "did:key:z6Mkk...xALi",
+  "aud": "did:web:web3.storage",
+  "att": [
+    {
+      can: "store/remove",
+      with: "did:key:zAl..1ce",
+      nb: {
+        link: "bag...7ldq",
+      }
+    }
+  ],
+  "prf": [{ "/": "bafy...prf1" }],
+  "exp": 1685602800,
+  "s": {
+    "/": {
+      "bytes": "7aEDQJbJqmyMxTxcK05XQKWfvxG+Tv+LWCJeE18RSMnciCZ/RQ21U75LA0uFSvIjdqnF5RaauZTE8mh2ZYMBBejdJQ4"
+    }
   }
 }
 ```
 
-#### Responses
 
-*Note*: This section is non-normative and subject to change, pending the [formalization of receipts and invocations][invocation-spec-pr].
+#### Store Remove Receipt
 
-Executing a `store/remove` invocation with a service provider should return a response object.
+Capability provider MUST issue signed receipt containing `StoreRemoveResult`.
 
-If a failure occurs, the response will have an `error` field with a value of `true`, and a `message` string field with details about the error.
+##### Store Remove Result
 
-On success, the response object will be empty.
+###### Store Remove Result IPLD Schema
 
-#### Implementations
-
-- @web3-storage/capabilities [store/remove validator](https://github.com/web3-storage/w3up/blob/5d52e447c14e7f7fd334e7ff575e032b7b0d89d7/packages/capabilities/src/store.js#L106)
-- @web3-storage/upload-api [store/remove method](https://github.com/web3-storage/w3up/blob/5d52e447c14e7f7fd334e7ff575e032b7b0d89d7/packages/upload-api/src/store/remove.js#L11)
+```ipldsch
+type StoreRemoveResult struct {
+  StoreRemoveSuccess   "ok"
+  StoreRemoveFailure   "error"
+} representation keyed
+```
 
-### `store/list`
+###### Store Remove Success
 
-> Obtain a list of stored CARs
+Capability provider MUST issue `StoreRemoveSuccess` after it has removed the archive from the space.
```

Comment:
```suggestion
Capability provider MUST issue `StoreRemoveSuccess` after it unlinked the archive from the space.
```
I think using unlined would be better, so that it does not get strictly understood as content won't be available anymore on the network